src/styles/scenes/PluginsStyle.js

```diff
@@ -60,6 +61,9 @@ export const styles = {
   },
   textBoxWrap: {
     /* justifyContent: 'center' */
+    height: scale(18 + 4 + 4 * 18),
+    // width minus left logo box minus padding
+    width: PLATFORM.deviceWidth - scale(50 + 16) - scale(40)
```

Comment:
Hard-coded dimensions based on device dimensions are a huge red flag! I will find another way to fix this!